docs/parameters.md

```diff
@@ -40,10 +40,11 @@ The following parameters are required in the YAML file describing any filter of
 | sky_brightness       |   float   | mag / arcsec^2^ | Average sky brightness computed for the survey and this filter. The moon conditions under which this number was computed will be given as a comment in the yaml file.                                                                                                                         |
 | full_exposure_time   | int/float |        s        | Average exposure time of the filter on the same spot in the sky over the course of the survey.                                                                                                                                                                                                |
 | psf_fwhm             |   float   |     arcsec      | Average full width at half-maximum (FWHM) of the point spread function (PSF) over the filter.                                                                                                                                                                                                 |
-| zeropoint            |   float   |       mag       | The zeropoint is the magnitude of an object that produces one electron per second on the detector. It is computed for a given filter, using the [`speclite`][speclite] library with a classical atmosphere, at the airmass given by the survey `zeropoint_airmass`.                           |
+| zeropoint            |   float   |       mag       | The zeropoint value is a reference magnitude computed as the magnitude in the AB system (see [this reference][abmag_ref]) collected in one second and on the survey effective area. It takes into account the filter bandwidth and a classical atmosphere for ground based surveys, using the [`speclite`][speclite] library, at the airmass given by the survey `zeropoint_airmass`.                           |
```

Comment:
I would say the "instrument effective area" not to be confused with the total area covered by the survey, even though I doubt that someone would misunderstand this.

Then I don't know if it is necessary to mention that speclite is not used for all surveys.

Reply:
I agree with @mpaillassa on these two points. Otherwise looks clear to me

Reply:
what do we do with the other surveys?

Reply:
For the other surveys we find other references (this includes CFHTLS and COSMOS).

Reply:
Yep, I will default to the references section then.